src/libraries/Common/src/Interop/OSX/Interop.libc.cs

```diff
@@ -44,5 +44,9 @@ internal static unsafe int fsetattrlist(SafeHandle handle, AttrList* attrList, v
                     handle.DangerousRelease();
             }
         }
+
+        [LibraryImport(Libraries.libc, EntryPoint = "clonefile", StringMarshalling = StringMarshalling.Utf8, SetLastError = true)]
+        internal static unsafe partial int clonefile(string from, string to, int flags);
```

Comment:
We try to match the native naming:
```suggestion
        internal static unsafe partial int clonefile(string src, string dst, int flags);
```